easy-random-core/src/main/java/org/jeasy/random/FieldPredicates.java

```diff
@@ -46,8 +46,10 @@
      * @param name pattern of the field name to check
      * @return Predicate to check that a field has a certain name pattern
      */
-    public static Predicate<Field> named(final String name) {
-        return field -> Pattern.compile(name).matcher(field.getName()).matches();
+    public static Predicate<Field> named(final String name)
+    {
```

Comment:
Please undo this formatting change. Thanks!

Reply:
done